src/nxdoc/nuxeo-server/administration/elasticsearch-setup.md

```diff
@@ -734,6 +760,40 @@ You can fine tune the indexing process using the following options:
     elasticsearch.reindex.bucketWriteSize=50
     ```
 
+### Re-index the repository using the Bulk Service
+
+Run a bulk command to re-index the repository, the command id is returned:
+```bash
+curl -s -X POST "<SERVER_URL>/nuxeo/site/automation/Elasticsearch.BulkIndex" -u Administrator:<PASSWORD> -H 'content-type: application/json+nxrequest' -d '{"params":{},"context":{}}'
```

Comment:
`application/json+nxrequest` is deprecated since 10.3 in favor of `application/json`.